docs/README.md

```diff
@@ -30,25 +30,25 @@ Usage:
 
 ```hcl
 provider "rubrik" {
-  node_ip     = "10.255.41.201"
+  node_ip     = "192.168.100.100"
   username    = "admin"
   password    = "RubrikTFDemo2019"
 }
 ```
 ### Environment variables
 
-You can provide your credentials via the `rubrik_cdm_node_ip`, `rubrik_cdm_username` and
-`rubrik_cdm_password`, environment variables, representing your Rubrik Node IP address, username
+You can provide your credentials via the `RUBRIK_CDM_NODE_IP`, `RUBRIK_CDM_USERNAME` and
+`RUBRIK_CDM_PASSWORD`, environment variables, representing your Rubrik Node IP address, username
 and password, respectively.
 
 ```hcl
 provider "rubrik" {}
 ```
 
 ```sh
-$ export rubrik_cdm_node_ip="10.255.41.201"
-$ export rubrik_cdm_username="admin"
-$ export rubrik_cdm_password="RubrikTFDemo2019"
+$ export RUBRIK_CDM_NODE_IP="10.255.41.201"
```

Comment:
Can we scrub this please?

Reply:
As per Infosec these examples can stay in the docs. I did update the example IP in commit [b21622f](https://github.com/rubrikinc/terraform-provider-rubrik/pull/76/commits/b21622fe81a45645641ebae950aab1b926c75889) to make it consistent with the rest of the docs.